dev/next/app/motion.tsx

```diff
@@ -0,0 +1,17 @@
+"use client"
```

Comment:
This should work without `"use client"` if we're importing from `motion/react-client`.

Otherwise you could just do `import { motion } from "motion/react"` like normal

Reply:
@healtheloper I think you're right, I'm happy for you to remove `motion.create` from the client export. Can I just double check then that you're happy with this PR as it is, other than that?

Reply:
@mattgperry Yes, thanks for confirming. I'll go ahead and remove `motion.create` from the client export.